packages/eslint-plugin-react-hooks/__tests__/ESLintRulesOfHooks-test.js

```diff
@@ -44,6 +44,25 @@ const tests = {
         }
       `,
     },
+    {
+      syntax: 'flow',
+      code: normalizeIndent`
+        // Component syntax
+        component Button() {
+          useHook();
+          return <div>Button!</div>;
+        }
+      `,
+    },
+    {
+      syntax: 'flow',
+      code: normalizeIndent`
+        // Component syntax
+        hook useSampleHook() {
+          useHook();
+        }
+      `,
+    },
```

Comment:
Could we add some tests to verify that the eslint rules also validate component/hook syntax? Like maybe an invalid test that shows the validations are still firing.

Reply:
Done